The minimum and maximum operators were never terribly portable and have
been dropped by GCC 4.2.
Index: langdrill-0.3/src/main.cc
===================================================================
--- langdrill-0.3.orig/src/main.cc	2007-07-15 12:52:38.000000000 +0100
+++ langdrill-0.3/src/main.cc	2007-07-15 12:54:01.000000000 +0100
@@ -460,9 +460,11 @@
   // keys. In such a case, we must take some precations
   setActiveButtons( keyTotalNr );
 
-  // is this minimum operator portable ??
-  // it should work with egcs at least !!
-  int cols = keyTotalNr <? atoi(_setup.quizzAnswerNr);
+  int cols;
+  if (keyTotalNr < atoi(_setup.quizzAnswerNr))
+    cols = keyTotalNr;
+  else
+    cols = atoi(_setup.quizzAnswerNr);
 
   int rows = keyTotalNr;
   int *set = new int[cols];
